pillar/nginx/chalkradio_redirect.sls

```diff
@@ -0,0 +1,34 @@
+{% set tlds = ['com', 'org', 'net'] %}
+
+nginx:
+    servers:
+    managed:
+      {% for tld in tlds %}
+      chalkradio-{{ tld }}-redirect:
+        enabled: True
+        config:
+          - server:
+              - server_name: chalkradiopodcast.{{ tld }}
+              - listen: 80
+              - listen: '443 ssl'
+              - listen: '[::]:80'
+              - listen: '[::]:443 ssl'
+              # The certificate uses subject alternative names.
+              - ssl_certificate: /etc/letsencrypt/live/amps-web.amps.ms.mit.edu/cert.pem
+              - ssl_certificate_key: /etc/letsencrypt/live/amps-web.amps.ms.mit.edu/privkey.pem
+              - ssl_stapling: 'on'
+              - ssl_stapling_verify: 'on'
+              - ssl_session_timeout: 1d
+              - ssl_session_tickets: 'off'
+              - ssl_protocols: 'TLSv1.2 TLSv1.3'
+              - ssl_ciphers: "TLS_AES_256_GCM_SHA384:TLS_CHACHA20_POLY1305_SHA256:TLS_AES_128_GCM_SHA256\
+                  :DHE-RSA-AES256-GCM-SHA384:DHE-RSA-AES128-GCM-SHA256:ECDHE-RSA-AES256-GCM-SHA384\
+                  :ECDHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES256-SHA256:DHE-RSA-AES128-SHA256\
+                  :ECDHE-RSA-AES256-SHA384:ECDHE-RSA-AES128-SHA256"
+              - ssl_prefer_server_ciphers: 'on'
+              - resolver: 1.1.1.1
+              - location /.well-known/:
+                - alias: /usr/share/nginx/html/.well-known/
+              - location /:
+                - return: 301 https://ocw.mit.edu/educator/chalk-radio-podcast/
```

Comment:
Do we want all routes on these domains to redirect specifically to that path, or do we want the path to be appended to that base?

Reply:
My understanding from the ZenDesk ticket is that they want all routes to redirect to this path. The use case is they announce the domain "chalktalkpodcast" on the podcast and the user gets to the this page on the OCW site.